src/App.js

```diff
@@ -234,8 +234,11 @@ class App extends React.Component {
         const canUpgradeOrg = repos.some(
           ({ appId, currentVersion, latestVersion }) =>
             isKnownRepo(appId) &&
-            currentVersion.version.split('.')[0] <
-              latestVersion.version.split('.')[0]
+            // If the installed app version is not a published version,
+            // also consider it for upgrades
+            (!currentVersion ||
```

Comment:
Does it mean that we want users of unpublished versions to “upgrade” by moving back to the latest published version of an app?

Reply:
Hmm good point.

The intention behind this was to notify these organizations when a new app upgrade comes out, but as we don't know which version their unpublished version is closest to, the actual behaviour of this would be to always annoy these organizations with the upgrade message—even when it's a downgrade back to the previous version.

Let's disregard organizations using unpublished versions from upgrades for now.

Reply:
Fixed in https://github.com/aragon/aragon/pull/1253/commits/30905b3ff4e937764827a3db1066e660d1cbd53d.